lib/access_lint/rails/engine.rb

```diff
@@ -5,7 +5,7 @@ class Engine < ::Rails::Engine
 
       initializer "access_lint-rails" do |app|
         app.config.assets.precompile << "accesslint.js"
-
+        app.config.filter_parameters << :accesslint
```

Comment:
That's great feedback, do you mean the `development.rb` for the host app or for the engine?